src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -431,25 +438,35 @@
     <Compile Include="Microsoft\Data\SqlClient\SqlColumnEncryptionCspProvider.cs" />
     <Compile Include="Microsoft\Data\SqlClient\SqlColumnEncryptionEnclaveProvider.cs" />
     <Compile Include="Microsoft\Data\SqlClient\SqlCommand.cs" />
-    <Compile Include="Microsoft\Data\SqlClient\SqlCommandBuilder.cs" />
+    <Compile Include="Microsoft\Data\SqlClient\SqlCommandBuilder.cs">
```

Comment:
Revert these and then hope we never see them again :weary:

Reply:
I added `DesignerCategory("")` to all of those items since it seemed to work with `SqlCommand` and made sure the SubType was still removed with an unload, reload and rebuild cycle.

Reply:
:v: